data/states.json

```diff
@@ -6,10 +6,8 @@
     "registration_type": "online",
     "translationKey": "Alabama",
     "type": "register",
-    "hp_link": "https://www.sos.alabama.gov/alabama-votes",
-    "registration_link": "https://www.alabamainteractive.org/sos/voter_registration/voterRegistrationWelcome.action",
-    "more_info_link": "https://www.sos.alabama.gov/alabama-votes/voter/register-to-vote",
-    "confirm_registration_link": "https://myinfo.alabamavotes.gov/VoterView/RegistrantSearch.do"
```

Comment:
We have a regression here. We lost all of the links for the register pages. Please add the links back into the states.json file and test to make sure the links on the state pages are working.

Reply:
I've added the missing links